spec/src/main/asciidoc/core/RelatedDocuments.adoc

```diff
@@ -1,137 +1,137 @@
 [[a9849]]
+[bibliography]
 == Related Documents
 
-Enterprise JavaBeans™, version 3.2. (this
+- [[[a9850, 1]]] Enterprise JavaBeans™, version 3.2. (this
```

Comment:
All of these spec references should be pointing at the Jakarta EE location now...  For example, https://jakarta.ee/specifications/enterprise-beans/3.2/